docs/references/openapi.yaml

```
@@ -762,6 +770,23 @@ paths:
         required: true
         content:
           application/pdf: {}
+          application/json:
+            schema:
+              properties:
+                files:
+                  type: array
+                  items:
+                    type: object
+                    properties:
+                      url:
+                        type: string
+                        format: url
+                        description: |
+                          DA URLs to the documents to sign.
```

```suggestion
                          DA URL to the documents to sign.
```
It's only one, even if it's an array of objects, right?